summingbird-storm/src/main/scala/com/twitter/summingbird/storm/BuildSummer.scala

```diff
@@ -118,8 +120,8 @@ object BuildSummer {
               tupleInCounter,
               tupleOutCounter,
               futurePool,
-              Compact(false),
-              CompactionSize(0))
+              Compact(doCompact.get),
```

Comment:
AsyncListSum takes Compact as parameter. AsyncListSum is defined in Algebird so can't change it's signature here.